AppCreationScripts/sample.json

```diff
@@ -12,15 +13,20 @@
   "AADApps": [
     {
       "Id": "service",
-      "Name": "TodoListService",
+      "Name": "TodoListService-Cert",
       "IsPublicClient": false,
+      "Kind":"WebApi",
+      "AvailableToOtherTenants": false,
       "HomePage": "https://localhost:44321/"
     },
     {
       "Id": "client",
-      "Name": "TodoListDaemon",
+      "Name": "TodoListDaemon-Cert",
       "Certificate": "CN=TodoListDaemonWithCert",
       "IsPublicClient": false,
+      "Kind":"Daemon",
+      "AvailableToOtherTenants": false,      
```

Comment:
See :) **My Org**


you might want to set **Audience** to **AzureADMyOrg**.  This will generate the right things